README.rst

```diff
@@ -0,0 +1,99 @@
+Introduction
+============
+
+.. image:: https://readthedocs.org/projects/adafruit-circuitpython-matrixportal/badge/?version=latest
+    :target: https://circuitpython.readthedocs.io/projects/matrixportal/en/latest/
+    :alt: Documentation Status
+
+.. image:: https://img.shields.io/discord/327254708534116352.svg
+    :target: https://discord.gg/nBQh6qu
```

Comment:
I don't think this invite link is active anymore. Looks like we need to update the cookiecutter.

Reply:
Ok, I can remove the link at least.

Reply:
https://github.com/adafruit/Adafruit_CircuitPython_AHTx0/pull/2/files

@dherrada Does cookiecutter need to be updated for the new discord link?

Reply:
Thanks for the correct link